src/google-maps/map-marker-clusterer/map-marker-clusterer.ts

```diff
@@ -177,6 +177,10 @@ export class MapMarkerClusterer implements OnInit, AfterContentInit, OnChanges,
   @Output()
   clusteringend: Observable<void> = this._eventManager.getLazyEmitter<void>('clusteringend');
 
+  /** Emits when a cluster has been clicked. */
+  @Output()
+  clusterClick: Observable<Cluster> = this._eventManager.getLazyEmitter<Cluster>('click');
```

Comment:
Not totally sure about the correct name here. In other places we've been using the `map` prefix (e.g. `mapClick` on the marker), but in this case the user is clicking a cluster within the clusterer, not the map. I'm open to changing the name.